libs/agno/agno/run/v2/workflow.py

```diff
@@ -432,62 +480,7 @@ def to_dict(self) -> Dict[str, Any]:
         if self.content and isinstance(self.content, BaseModel):
             _dict["content"] = self.content.model_dump(exclude_none=True)
 
-        return _dict
-
-    def to_json(self) -> str:
-        import json
-
-        _dict = self.to_dict()
-        return json.dumps(_dict, indent=2)
-
-    @classmethod
-    def from_dict(cls, data: Dict[str, Any]) -> "WorkflowRunResponse":
```

Comment:
I think you'll need this to go from the DB to the loaded state? So for `read_from_storage` ?

Reply:
good catch